packages/app/src/debug/DebugRunDetailedView.vue

```diff
@@ -0,0 +1,253 @@
+<template>
+  <div class="border border-indigo-100 rounded">
+    <div
+      class="bg-indigo-50 p-12px flex items-center"
+      data-cy="debug-detailed-header"
+    >
+      <div
+        class="flex items-center w-full justify-between"
+      >
+        <div class="flex items-center">
+          <button
+            class="flex items-center p-6px mr-8px"
+            data-cy="debug-toggle"
+            @click="toggleRuns"
+          >
+            <IconChevronDownLarge
+              v-if="showRuns"
+              stroke-color="indigo-400"
+            />
+            <IconChevronRightLarge
+              v-else
+              stroke-color="indigo-400"
+            />
+          </button>
+
+          <LightText v-if="latestIsCurrentlySelected">
+            You are on the most recent run
+          </LightText>
+
+          <template v-else-if="latest?.status && latest.runNumber">
+            <DebugRunNumber
+              :status="latest.status"
+              :value="latest.runNumber"
+              class="mr-8px"
+            />
+            <DebugResults
+              v-if="latest"
+              :gql="latest"
+            />
+            <span class="pl-16px">{{ latest.commitInfo?.summary }}</span>
+            <Dot />{{ specsCompleted(latest) }}
+          </template>
+        </div>
+        <Button
+          v-if="!latestIsCurrentlySelected"
+          data-cy="switch-to-latest"
+        >
```

Comment:
```suggestion
        <Button
          v-if="!latestIsCurrentlySelected"
          data-cy="switch-to-latest"
          @click="$event => changeRun(latest!)"
        >
```
Just adding this one click event should get this button working